lib/mini_profiler/profiler.rb

```diff
@@ -809,10 +814,14 @@ def take_snapshot(env, start)
         )
         custom_fields = MiniProfiler.get_snapshot_custom_fields
         page_struct[:custom_fields] = custom_fields if custom_fields
-        @storage.push_snapshot(
-          page_struct,
-          @config
-        )
+        if Rack::MiniProfiler.snapshots_transporter?
```

Comment:
@SamSaffron should we skip storage if transportation is configured? This line will skip storage if transportation is configured, but it shouldn't be difficult to change it so we store snapshots and ship them. A downside of skipping storage is that some snapshots might get lost if ruby process is killed/restarted before buffered snapshots are shipped.

Reply:
Yeah I think we can skip for now if transport is configured, seems reasonable.